### ISSUE TITLE
[ui fluid container] - view on mobile devices

### ISSUE THREAD

**Issue by wojciech-mocarski** [Semantic-Org/Semantic-UI#2895]
**Steps to Reproduce**
1. Use:
```html
<div class="ui fluid container">
  Use long text .... (or 100% width wide other elements like lists or menu)
</div>
```
or check http://jsfiddle.net/9udrtL2w/
2. Narrow your web browser to mobile device's size
3. Check both margins of the text. Left margin moves ~10 pixels to the right direction and text (or element) is partially hidden under the browser scroll bar or right margin.

**Expected**
The container should show whole page content visible

**Result**
The Container does not show correct view (right part of text or element is hidden)

**Comment by jlukic** [comment]
Related to missing `margin-left/right` on `stackable grid` should be solved with same fix as #2898

**Comment by jlukic** [comment]
Fixed in `2.1`
http://jsfiddle.net/qvvegu5v/
